actions/Create DB User/code.js

```diff
@@ -0,0 +1,26 @@
+const fetch = require('node-fetch')
+
+exports.onExecutePostLogin = async (event, api) => {
+  // 1.  
+  const SECRET = event.secrets.AUTH0_HOOK_SECRET
```

Comment:
This code looks to support object destructuring https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

It would be nicer to write:

```diff
-  const SECRET = event.secrets.AUTH0_HOOK_SECRET
+  const { AUTH0_HOOK_SECRET: secret } = event.secrets
```

and below 


```diff
-     body: JSON.stringify({ email, secret: SECRET }),
+    body: JSON.stringify({ email, secret }),
```